go/stats/statsd/statsd.go

```diff
@@ -229,3 +229,15 @@ func (sb StatsBackend) PushAll() error {
 	}
 	return nil
 }
+
+// PushOne pushes the single provided metric.
```

Comment:
This is new to satisfy changes to the `PushBackend` interface, and likely nothing in the codebase reaches here.